### PR TITLE
Add Course

### PR DESCRIPTION
**Pull request by emikate**
Admin can now access a new page 'Add Course' from the dashboard (addCourse.php) where the form to add a course repeats dynamically by clicking on 'Add Another Course'. Once forms are completed, the Create Courses option adds them to the database and calendar.

### CODE REVIEW

**Comment by AlexKeehan** [comment]
Code looks good!

**Comment by tnoor99** [comment]
Hello Emily, I reviewed your addCourse.php for this code review assignment. Entire code has effective commenting. Comments are placed in correct places, before a new section of code begins. Comments are also informative, short, and to the point. Variables are descriptive, meaningful names and easy to comprehend. Indentation is consistent, making is easier to read overall. Additionally, code demonstrates effective and efficient programming, making it easier to change and maintain overtime if needed. Lastly, great job on completing on your assigned task and having it properly functioning. 

**Additional Specifics**

- Proper implementation of updating additional files such as header.php to ensure your new features of addCourse are effectively implemented.
- In Index.php, proper format was utilized when adding add course on to the dashboard for the front end. This allowed for the code throughout index.php to stay consistent.

File: addCourse.php [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+            $description, 
+            $location, 
+            $capacity
+        ];
+
+        $courses[] = $courseArgs;
+    }
+
+    // Create the new courses
+    foreach ($courses as $courseArgs) {
```
Comment (tnoor99) [inline addCourse.php:79]: Effective way to creating a new course, short, simple, and straight to the point.

File: addCourse.php [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+        $trainer = $args['new_course_trainer'];
+        $description = $args['new_course_description'];
+        $location = $args['new_course_location'];
+        $capacity = intval($args['new_course_capacity']);
+        $eventId = $args['event_id']; 
+
+        // Check if any required fields are missing or invalid for each course
+        if (empty($courseName) || empty($startTime) || empty($endTime) || empty($date) || empty($trainer) || empty($description) || empty($location) || empty($eventId) || $capacity < 1 || $capacity > 20) {
+            // Redirect back to the form with an error message
+            header("Location: addCourse.php?error=missing_fields");
```
Comment (tnoor99) [inline addCourse.php:50]: Effective use of comments. Provides an easy description to help reader understand.

**Pull request merged**